aloha-core/src/main/scala/com/eharmony/aloha/reflect/RefInfoOps.scala

```diff
@@ -159,6 +169,36 @@ object RefInfoOps extends RefInfoOps[RefInfo] with EitherHelpers {
   def typeParams[A](implicit a: RefInfo[A]): List[RefInfo[_]] = a.typeArguments
 
   /**
+    * Attempt to determine if `possibleIterable` is an `Iterable[A]`.
+    *
+    * Covers only simple one type variable instances in Predef.  This is kind of a stopgap solution because the
+    * subtype operator `<:<` doesn't really work well for Manifests.  This is an issue because currently, Aloha uses
+    * Manifests rather than `TypeTag`s for type checking.  Comments in ClassManifestDeprecatedApis says:
+    *
+    * "''this part is wrong for punting unless the rhs has no type arguments, but it's better
```

Comment:
No.  This will never be fixed.  In fact, `Manifest` which is the backing type of `RefInfo` has been deprecated since 2.10 and will most likely be removed.  They keep toying with it and threatened to remove it in 2.12 but kept it because people are still using it. `RefInfo` should eventually become `ClassTag` I think but there are cases when a `TypeTag` is more useful (like in sub-type checking).